Source/WebCore/rendering/RenderObject.h

```diff
@@ -402,6 +402,8 @@ class RenderObject : public CachedImageClient, public CanMakeCheckedPtr {
 
     bool everHadLayout() const { return m_bitfields.everHadLayout(); }
 
+    ScrollAnchoringController* scrollAnchoringController() const;
```

Comment:
Having this here makes it sound like each RenderObject has one, but this is really a "find one by traversing ancestors", so the name should reflect that.

It would even be better as a static function: `static ScrollAnchoringController* findScrollAnchoringControllerForRenderer(const RenderObject&)`.

Reply:
Fixed.